packages/apputils/src/vdom.ts

```diff
@@ -95,8 +95,15 @@ export abstract class ReactWidget extends Widget {
  * An abstract ReactWidget with a model.
  */
 export abstract class VDomRenderer<
-  T extends VDomRenderer.IModel | null
+  T extends VDomRenderer.IModel | null = null
 > extends ReactWidget {
+  /**
+   * Create a new VDomRenderer
+   */
+  constructor(model: T extends null ? void : T) {
```

Comment:
This commit changes the `VDomRenderer` to take the model as a constructor arg, in order to make it non-nullable. This was an easy fit here, but might give less flexibility for 3rd party renderers?

Reply:
The reason not to have it in the constructor was because setting the model calls override-able methods, which would not call overrridden versions of those methods in the subclass.

Reply:
No test failures 😉

Reply:
Do you want to revert this behavior then? Or perhaps have the constructor call a protected setter method (or something similar)?